### DISCUSSION TITLE
窗帘全开时显示打开99%

### DISCUSSION THREAD

**Discussion by Decwind** [XiaoMi/ha_xiaomi_home#423] [Ideas]
窗帘全开时显示打开99%，状态未与米家完全同步。
![IMG_0881](https://github.com/user-attachments/assets/09389560-1651-407e-9117-65cb8d754d3f)

**Comment by Decwind** [comment]
![Uploading IMG_0881.png…]()

**Reply by JimmyKmi** [reply]
图片上传失败
